src/resmom/mom_hook_func.c

```diff
@@ -572,7 +572,10 @@ copy_file_and_set_owner(char *src_file, char *dest_file, job *pjob) {
 		"Administrators",
```

Comment:
Please note src_file and dest_file here:
https://github.com/PBSPro/pbspro/blob/f84fe1585683331d08711c94ff02a11087b80f50/src/resmom/mom_hook_func.c#L528